ext/mysql2/result.h

```diff
@@ -1,5 +1,6 @@
 #ifndef MYSQL2_RESULT_H
 #define MYSQL2_RESULT_H
+#include <stdbool.h>
```

Comment:
I found it! While it looks dated at this point, the assumptions are stated here: https://bugs.ruby-lang.org/projects/ruby-trunk/wiki/Assumptions